purchase_discount/__manifest__.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 # © 2004-2009 Tiny SPRL (<http://tiny.be>).
-# © 2014-2015 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# © 2014-2017 Tecnativa - Pedro M. Baeza
```

Comment:
Not blocking, just questioning - can you copyright something with an entity that didn't exist on the data the copyright started?

Reply:
Well, it was me also before (as freelance), so I prefer to simplify it.